bioblend/_tests/pytest_galaxy_test_wrapper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import sys
+
+try:
+    import pytest
```

Comment:
So I feel like all the Galaxy projects should be slowly transitioning to pytest - it is more modern and better supported Python testing framework. Compare https://github.com/pytest-dev/pytest and https://github.com/nose-devs/nose for instance - it seems like pytest is more actively developed. Since this would expose the underlying testing options - I thought it would be better than nose.

I'd be happy to switch the rest of the bioblend testing stack to use pytest - it should be done anyway. I need to do the same for pulsar, planemo, and ultimately Galaxy.